homebox/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+
+  app_proxy:
+    environment:
+      APP_HOST: homebox_server_1
+      APP_PORT: 7745
+    
+  server:
+    image: ghcr.io/hay-kot/homebox:v0.9.1@sha256:93eb59c89dc59cf2a6bd3943828ecc30f5a8c8f34fd844c79f54801e914ea64e
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
```

Comment:
It doesn't look like the homebox image supports these environment variables. To make sure that the data directory gets created with the correct non-root user permissions you can add `user: 1000:1000` above or below the `environment` section. See this example from the Lightning  Node app:

https://github.com/getumbrel/umbrel-apps/blob/6841e3b74b7cf328ce4e8e65be7f5b140533440f/lightning/docker-compose.yml#L43-L48